pkg/logstructured/logstructured.go

```diff
@@ -198,15 +198,16 @@ func (l *LogStructured) List(ctx context.Context, prefix, startKey string, limit
 	return rev, kvs, nil
 }
 
-func (l *LogStructured) Count(ctx context.Context, prefix string) (revRet int64, count int64, err error) {
+func (l *LogStructured) Count(ctx context.Context, prefix string, revision int64) (revRet int64, count int64, err error) {
 	defer func() {
-		logrus.Tracef("COUNT %s => rev=%d, count=%d, err=%v", prefix, revRet, count, err)
+		logrus.Tracef("COUNT %s, rev=%d => rev=%d, count=%d, err=%v", prefix, revision, revRet, count, err)
 	}()
-	rev, count, err := l.log.Count(ctx, prefix)
+	rev, count, err := l.log.Count(ctx, prefix, revision)
 	if err != nil {
 		return 0, 0, err
 	}
 
+	// FIXME
```

Comment:
Context?

Reply:
lol I did fix it, but I forgot to remove the note.